CHANGELOG.md

```diff
@@ -151,7 +151,7 @@ Please verify the usage of `sentry.Flush` in your code base.
 
 ## v0.3.0
 
-- feat: Retry event marshalling without contextual data if the first pass fails
+- feat: Retry event marshaling without contextual data if the first pass fails
```

Comment:
It's double `L` :P

https://en.wikipedia.org/wiki/Marshalling_(computer_science)

Reply:
AFAICT this is UK vs US spelling: https://wikidiff.com/marshaled/marshalled.
Occurrences in [encoding/json](https://golang.org/pkg/encoding/json/) all use single L.